e2e/pageobjects/openshift/OcpWebConsolePage.ts

```diff
@@ -18,8 +18,13 @@ import { Logger } from '../../utils/Logger';
 @injectable()
 export class OcpWebConsolePage {
 
-    private static readonly CHE_OPERATOR_LOGO_NAME: string = '//h1[text()=\'Eclipse Che\']';
-    private static readonly ECLIPSE_CHE_PREFIX_URL: string = 'che-';
+    private static readonly OPERATOR_LOGO_NAME: string = `//h1[text()='${TestConstants.TS_SELENIUM_OPERATOR_LOGO_NAME}']`;
```

Comment:
It would be great to point which kind of locator is used (xpath, css ...) . Here and in all other places.